src/app-layout/visual-refresh-toolbar/split-panel/index.tsx

```diff
@@ -12,17 +12,19 @@ interface AppLayoutSplitPanelDrawerSideImplementationProps {
   appLayoutInternals: AppLayoutInternals;
   splitPanelInternals: SplitPanelProviderProps;
   children: React.ReactNode;
+  animationDisabled?: boolean;
```

Comment:
Let's avoid adding more properties on these widget-non-widget boundaries. If we need to pass something down, it should go via `appLayoutInternals` (or ideally find a way to avoid this, for example, applying a class name on a DOM element above that)